too_simple_server/tasks/main.yml

```diff
@@ -4,7 +4,7 @@
   register: LATEST_VERSION
 
 - name: Check active version of too_simple_server
-  shell: "{{ server|quote }} --version" # noqa 301 305
+  command: "{{ server|quote }} --version" # noqa 301 305
```

Comment:
If we use command we will not use quote filter

Reply:
shell is't good too, are there other way?

Reply:
i mean we don't need to use `quote` filter if we moved to command